Confus/ConfusServer/Audio/PlayerAudioEmitter.cpp

```diff
@@ -0,0 +1,50 @@
+#include "PlayerAudioEmitter.h"
+
+namespace ConfusServer
+{
+    namespace Audio
+    {
+        PlayerAudioEmitter::PlayerAudioEmitter(irr::scene::IAnimatedMeshSceneNode* a_AttachedPlayer) : m_AttachedPlayer(a_AttachedPlayer)
+        {
+            createAudioSources();
+        }
+
+        PlayerAudioEmitter::~PlayerAudioEmitter()
+        {
+            delete(m_AudioSourceFootstep1);
+            delete(m_AudioSourceFootstep2);
+            delete(m_AudioSourceFootstep3);
+            delete(m_AttachedPlayer);
```

Comment:
We're going to need a system to keep the server project updated with the confus project. @Athosvk, do you have an idea for this?